dotnet/samples/KernelSyntaxExamples/Example19_Qdrant.cs

```diff
@@ -24,7 +24,7 @@ public static async Task RunAsync()
             .Build();
 
         Console.WriteLine("== Printing Collections in DB ==");
-        var collections = memoryStore.GetCollectionsAsync();
+        var collections = memoryStore.GetCollectionsAsync().ConfigureAwait(false);
```

Comment:
These ConfigureAwaits shouldn't be required as KernelSyntaxExamples is an app, not an SDK. I see they were being enforced because the .editorconfig that enabled the rule was in /dotnet. I moved it to /dotnet/src, so this project should now be free of its tyranny. Please sync run again and, if I'm not mistaken, these ConfigureAwaits can now be reverted!

Reply:
reverted with [that commit](https://github.com/microsoft/semantic-kernel/pull/1914/commits/bfeeef7ff9cf6c18e312fb833cbca5c7951f2138)